front-end/src/Components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Button = ({text, onClick, className}) =>{
+
+	return(
+		<button className={className} onClick={onClick}>
+			{text}
+		</button>
+	)
+}
+
+
+export default Button
```

Comment:
Let's not try to re-invent the wheel on this one. Bootstrap provides a component called Button as well, so I'd recommend just using their Button component for the "Register" button, and use a <Link> for the login link.

Please delete this component and refactor to leverage Bootstrap's code.

Reply:
The components for button was left over, I did leverage the button on the homepage from "styled components" so should i just delete the button component?

Reply:
I don't want to use styled components for this button. Please use the <Button> component that's supplied by React-Bootstrap

Reply:
i have attempted to change the button to use button from react